ci-operator/step-registry/quay-tests/test-quay-e2e/quay-tests-test-quay-e2e-commands.sh

```diff
@@ -16,6 +16,7 @@ mkdir -p $ARTIFACT_DIR
 
 function copyArtifacts {
     cp -r ./cypress/results/* $ARTIFACT_DIR && cp -r ./cypress/videos/smoke/* $ARTIFACT_DIR
+    #TODO: Find any XMLS generated and preprend "junit_" to the filename.
```

Comment:
In order for tests within the XML to be visible from the TestGrid dashboard automatically the syntax `junit_` needs to be prepended to the XML file name. The artifacts for the test that ran this past monday is[ here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-quay-quay-tests-master-ocp-414-quay38-quay-e2e-tests-quay38-ocp414-lp-interop/1663062757404053504/artifacts/quay-e2e-tests-quay38-ocp414-lp-interop/quay-tests-test-quay-e2e/artifacts/) 

If we configure this to change the filename from `quay_automation_testiong_report-4f2491ad7c1e96fb4ec891df158fe636.xml` to `junit_quay_automation_testiong_report-4f2491ad7c1e96fb4ec891df158fe636.xml`

then that should be all that is needed.

Reply:
@ascerra OK, we can update our code to add this prefix "junit_"